### DISCUSSION TITLE
Comments by line for change reviewers

### DISCUSSION THREAD

**Discussion by gedalyahreback** [GitbookIO/community#745] [Feature Requests]
It's possible to add comments by line or section if writing content, but not if reviewing it. This makes feedback harder. I feel like this should be a priority above other feature requests.

**Comment by SamyPesse** [comment]
The timing is perfect, we just ship this improvement! You can now comment when viewing changes in a change request:


https://github.com/user-attachments/assets/da1f29c1-9661-4edb-a296-2850dcab3c03
